aws/rapids_ec2_mnmg.ipynb

```diff
@@ -0,0 +1,456 @@
+{
```

Comment:
<ol><li>fit an RF model..</li><li>Since read and clean the data and add features are headings should the others bullet points be headings as well later in the notebook? May be reduce all of them to heading type 3 or heading type 4 instead of 2?</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cloud-ml-examples/pull/146/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>